ui/pages/access-control/index.js

```diff
@@ -0,0 +1,760 @@
+                {resourcesOptions?.length > 0 && (
+                  <div className='relative'>
+                    <Combobox.Options className=' absolute z-50 mt-2 max-h-60 w-full origin-top-right divide-y divide-gray-100 overflow-auto rounded-md bg-white text-xs shadow-lg shadow-gray-300/20 ring-1 ring-black ring-opacity-5 focus:outline-none'>
+                      {resourcesOptions?.map(f => (
+                        <Combobox.Option
+                          key={f.id}
+                          value={f}
+                          className={({ active }) =>
+                            `relative cursor-default select-none py-[7px] px-3 ${
+                              active ? 'bg-gray-50' : ''
+                            }`
+                          }
+                        >
+                          <div className='flex flex-row'>
+                            <div className='flex min-w-0 flex-1 flex-col'>
+                              <div className='flex justify-between py-0.5 font-medium'>
+                                <span className='truncate' title={f.name}>
+                                  {f.name}
+                                </span>
+                                {selectedResource &&
+                                  selectedResource.id === f.id && (
+                                    <CheckIcon
+                                      data-testid='selected-icon'
+                                      className='h-3 w-3 stroke-1 text-gray-600'
+                                      aria-hidden='true'
+                                    />
+                                  )}
+                              </div>
+                              <div className='text-3xs text-gray-500'>
+                                {f.kind}
+                              </div>
+                            </div>
+                          </div>
+                        </Combobox.Option>
+                      ))}
+                    </Combobox.Options>
+                  </div>
+                )}
+                <Combobox.Button className='hidden' ref={resourceButton} />
+              </Combobox>
+            </div>
+            {selectedResource?.kind !== 'ssh' && (
+              <>
+                {selectedResource !== null && (
+                  <div className='relative mt-4 space-y-1'>
+                    <label className='text-2xs font-medium text-gray-700'>
+                      Namespaces (optional)
+                    </label>
+                    <Listbox
+                      value={selectedNamespaces}
+                      onChange={v => {
+                        setError({})
+                        if (v.includes(OPTION_SELECT_ALL)) {
+                          if (
+                            selectedNamespaces.length !==
+                            selectedResource?.resources?.length
+                          ) {
+                            if (selectedResource !== undefined) {
```

Comment:
```suggestion
                            if (selectedResource) {
```

We _may_ be able to just check if this is truth-y